bookkeeper-server/src/test/java/org/apache/bookkeeper/test/BookieZKExpireTest.java

```diff
@@ -73,35 +74,26 @@ conf, new TestBookieImpl(conf),
                     NullStatsLogger.INSTANCE, UnpooledByteBufAllocator.DEFAULT,
                     new MockUncleanShutdownDetection());
             server.start();
-
-            int secondsToWait = 5;
-            while (!server.isRunning()) {
-                Thread.sleep(1000);
-                if (secondsToWait-- <= 0) {
-                    fail("Bookie never started");
-                }
-            }
+            BookieServer finalServer = server;
+            await().atMost(5, SECONDS).until(finalServer::isRunning);
             Thread sendthread = null;
             threadCount = Thread.activeCount();
             threads = new Thread[threadCount * 2];
             threadCount = Thread.enumerate(threads);
             for (int i = 0; i < threadCount; i++) {
-                if (threads[i].getName().indexOf("SendThread") != -1
+                if (threads[i].getName().contains("SendThread")
                         && !threadset.contains(threads[i])) {
                     sendthread = threads[i];
                     break;
                 }
             }
             assertNotNull("Send thread not found", sendthread);
-
             sendthread.suspend();
-            Thread.sleep(2 * conf.getZkTimeout());
```

Comment:
Looks like this is part of the testing. It wants to resume the send thread after 2* zkTimeout to test the bookie should still working even if the zk session is timeout.